tests/rust-integration-tests/integration_test/src/tests/hooks/hooks_test.rs

```diff
@@ -1,20 +1,27 @@
 use anyhow::anyhow;
 use oci_spec::runtime::{Hook, HookBuilder, HooksBuilder, ProcessBuilder, Spec, SpecBuilder};
-use std::{fs::File, io::Read};
+use std::{fs::File, io::Read, path::PathBuf};
 use test_framework::{Test, TestGroup, TestResult};
 
 use crate::utils::{
     create_container, delete_container, generate_uuid, prepare_bundle, set_config,
     test_utils::start_container,
 };
 
+fn absolute_path(path: &str) -> PathBuf {
+    let pwd = std::env::current_dir().unwrap();
+    pwd.join(path)
+}
+
 fn write_log_hook(content: &str) -> Hook {
+    let output = absolute_path("./output");
```

Comment:
How about using `std::fs::canonicalize` instead of `absolute_path`?
https://doc.rust-lang.org/std/fs/fn.canonicalize.html#